consts/consts.go

```diff
@@ -318,6 +319,9 @@ const ES_RESULT_TYPE_TAGS = "tags"
 const ES_RESULT_TYPE_BLOG_POSTS = "posts"
 const ES_RESULT_TYPE_TWEETS = "tweets"
 
+// Result of many tweets in one hit
+const SEARCH_RESULT_TWEETS_MANY = "tweets_many"
```

Comment:
Is this the same constant from the other PR?

Reply:
yes